src/include/timestamp_inline.h

```diff
@@ -91,25 +102,25 @@
     } while (0)
 
 /* Copy the start values of a time window from another time window. */
-#define WT_TIME_WINDOW_COPY_START(dest, source)                \
-    do {                                                       \
-        (dest)->durable_start_ts = (source)->durable_start_ts; \
-        (dest)->start_ts = (source)->start_ts;                 \
-        (dest)->start_txn = (source)->start_txn;               \
-        (dest)->prepare = (source)->prepare;                   \
-        (dest)->prepare_ts = (source)->prepare_ts;             \
-        (dest)->prepared_id = (source)->prepared_id;           \
+#define WT_TIME_WINDOW_COPY_START(dest, source)                  \
+    do {                                                         \
+        (dest)->durable_start_ts = (source)->durable_start_ts;   \
+        (dest)->start_ts = (source)->start_ts;                   \
+        (dest)->start_txn = (source)->start_txn;                 \
+        (dest)->prepare = (source)->prepare;                     \
+        (dest)->start_prepare_ts = (source)->start_prepare_ts;   \
+        (dest)->start_prepared_id = (source)->start_prepared_id; \
     } while (0)
 
 /* Copy the stop values of a time window from another time window. */
-#define WT_TIME_WINDOW_COPY_STOP(dest, source)               \
-    do {                                                     \
-        (dest)->durable_stop_ts = (source)->durable_stop_ts; \
-        (dest)->stop_ts = (source)->stop_ts;                 \
-        (dest)->stop_txn = (source)->stop_txn;               \
-        (dest)->prepare = (source)->prepare;                 \
-        (dest)->prepare_ts = (source)->prepare_ts;           \
-        (dest)->prepared_id = (source)->prepared_id;         \
+#define WT_TIME_WINDOW_COPY_STOP(dest, source)                 \
+    do {                                                       \
+        (dest)->durable_stop_ts = (source)->durable_stop_ts;   \
+        (dest)->stop_ts = (source)->stop_ts;                   \
+        (dest)->stop_txn = (source)->stop_txn;                 \
+        (dest)->prepare = (source)->prepare;                   \
```

Comment:
Here's another reason I think we should separate the prepare flag as well. It is not clear whether the prepare flag is for the start or the stop.

Reply:
We can leave the prepare flag here for now to keep the attached storage version. But we can remove that when we align the prepared logic with the old and new versions.

Reply:
Yes, the reason why I didn't split the prepare flag is because i find it a bit redundant. We can just check if `WT_TIME_WINDOW_HAS_STOP_PREPARE` and `WT_TIME_WINDOW_HAS_START_PREPARE` exist, and even if you have the flags you still have to check them anyway. 

Couldn't remove the prepare flag completely because this change is quite a bit already, so yes can do the cleanup when we align the code for both configs